code/modules/food_and_drinks/kitchen_machinery/gibber.dm

```diff
@@ -331,15 +343,22 @@
 			lturf = T
 
 	for(var/mob/living/carbon/human/H in lturf)
-		if(istype(H) && H.loc == lturf)
+		if(istype(H) && H.loc == lturf && !(locate(/obj/machinery/gibber/autogibber) in H.interaction_queue))
+			target_found = 1
+			H.interaction_queue += src
```

Comment:
This is so that multiple autogibbers on the same tile grab different targets instead of all ganging up on the first nerd, not so that it'll grab anyone else on the tile if the first nerd left.

Granted, adding a non-simple var on a type path as global as `/atom` is not a great idea, true. I also was thinking this would be useful for cleanbots - so that they would be able to divide work on messes among multiple bots more efficiently, instead of all piling in on one tile as I've observed them do before. But yes, adding stuff to the `atom` path is best avoided, particularly more complex var types such as lists